hq/app/views/snyk/snyk.scala.html

```diff
@@ -0,0 +1,56 @@
+@import logic.SnykDisplay
+@import logic._
+@import model._
+
+@(results: List[SnykProjectIssues])(implicit assets: AssetsFinder)
+
+@main(Nil) { @* Header *@
+    <div class="container">
+        <div class="row">
+            <h1 class="header center-on-small-only grey-text text-lighten-5">Security HQ</h1>
+            <div class="col s12 m9">
+                <h4 class="grey-text text-darken-4 center-on-small-only">Snyk Results</h4>
+            </div>
+        </div>
+    </div>
+
+} { @* Main content *@
+
+    <div class="container">
+
+        <table class="striped responsive-table filterable-table">
+            <thead>
+                <tr>
+                    <th>Project Name</th>
+                    <th>High</th>
+                    <th>Medium</th>
+                    <th>Low</th>
+                </tr>
+            </thead>
+
+            <tbody>
+            @for(project <- results) {
+                <tr>
+                    <td>
+                        <a class="black-text" href="https://snyk.io/org/guardian/project/@project.id/">@project.name</a>
```

Comment:
I have also not loaded up the UI (yet). However black links will be bad for a11y and discoverability.

I would make an exception if this is a card (as with the index page) where the entire card is clickable and therefore we may want black text on the card (see usage of `hp__link`)

If the layout is going to be similar to the IAM page, maybe that design pattern could be used?

<img width="1154" alt="picture 168" src="https://user-images.githubusercontent.com/8607683/35390526-afd20870-01d3-11e8-91c1-97baa70f0d01.png">

Reply:
Opinionated (IANAUX/D):

Black text on cards and within nav is okay (and is the current pattern), since most users will expect those areas to be clickable. However, within a table or body text, links should be visually distinct (at least a different colour, ideally text decoration).

Regarding accessibility: 

> To aid discoverability all links must be made self-evident with their visual style

http://www.bbc.co.uk/guidelines/futuremedia/accessibility/html/control-styles.shtml